styles/designs/carnival/parts/_table-components.scss

```diff
@@ -246,6 +246,14 @@ $Table__Data--NotLastChild: (
     )
 );
 
+$Table__Data--NotFirstChildWithRowspan: (
+    _name: "TableDataNotFirstChildWithRowspan",
+    _subselector: ' > td[rowspan="2"]:not(:only-of-type):not(:first-child)',
+    _properties: (
+        border-left: '0.025rem solid black',
+    )
+);
+
```

Comment:
I fixed it using an old approach, since Jennifer is already working on a basic table refactoring.